conversation/ui
# Add Typography docs

This PR adds documentation for the `<Typography>` component.

## How to test

* Run `make storybook`
* Open `localhost:9001` and check it out

![screenshot_2018-12-11 storybook](https://user-images.githubusercontent.com/3614/49778021-2cc6ac80-fd57-11e8-80c6-f169dd085d35.png)
